packages/pds-ember/addon/components/pds/app/body/index.hbs

```diff
@@ -1,7 +1,6 @@
 <main
   class="pds-app__body"
   id="content"
-  role="main"
```

Comment:
new template lint rules called this out as redundant on account of this already being a `<main>`